### PR TITLE
Support hex encoding output

### PR DESCRIPTION
**Pull request by huylenq**

### CODE REVIEW

File: src/main/java/org/embulk/filter/encrypt/EncryptFilterPlugin.java [hunk 1]
```diff
@@ -106,12 +106,58 @@ public String toString()
...
+        }
+    }
+
     public interface PluginTask
             extends Task
     {
         @Config("algorithm")
         public Algorithm getAlgorithm();
 
+        @Config("format")
```
Comment (dmikurube) [inline src/main/java/org/embulk/filter/encrypt/EncryptFilterPlugin.java]: The name `format` sounds a bit different from what it does. It is a kind of encoding, not just formatting.

What do you think `output_encoding` or like that?

File: README.md [hunk 2]
```diff
@@ -23,7 +23,8 @@ You can apply encryption to password column and get following outputs:
 - **algorithm**: encryption algorithm (see below) (enum, required)
 - **column_names**: names of string columns to encrypt (array of string, required)
 - **key_hex**: encryption key (string, required)
-- **iv_hex**: encyrption initialization vector (string, required if mode of the algorithm is CBC)
+- **iv_hex**: encryption initialization vector (string, required if mode of the algorithm is CBC)
+- **format**: encoding of the encrypted value, can be either "base64" or "hex" (base16)
```
Comment (dmikurube) [inline README.md]: Same as the comment of mine in the code.

File: README.md [hunk 3]
```diff
@@ -120,6 +121,7 @@ filters:
     column_names: [password, ip]
     key_hex: 098F6BCD4621D373CADE4E832627B4F60A9172716AE6428409885B8B829CCB05
     iv_hex: C9DD4BB33B827EB1FBA1B16A0074D460
+    format: hex
```
Comment (dmikurube) [inline README.md]: ditto.

**Review by dmikurube — COMMENTED** [review]
Basically looks good. Just commented about naming.

**Review by dmikurube — COMMENTED** [review]
And, please add unit test(s) for this feature.

File: src/main/java/org/embulk/filter/encrypt/EncryptFilterPlugin.java (hunk 1 above)
Comment (huylenq) [reply src/main/java/org/embulk/filter/encrypt/EncryptFilterPlugin.java]: I do agree

**Comment by huylenq** [comment]
Unit tests added, also cover existing implementation.

**Review by dmikurube — APPROVED** [review]
Looks good to me, but please confirm with @kietdo360 as well.

**Comment by huylenq** [comment]
Also changed the way `key_hex` is required for more meaningful error message. @kietdo360 could you take a look?

**Review by kietdo360 — APPROVED** [review]
LGTM

**Pull request merged**